tests/treatment/checkWhenNoTreatment.test.js

```diff
@@ -28,436 +28,455 @@ const treatment = require('../constructDummyData').getSchemaDummy('treatment');
-        [
-            'outcome_of_treatment is submitted when no treatment was given',
-            false,
-            loadObjects(treatment, {
-                outcome_of_treatment: 'treatment completed as prescribed',
-                treatment_type: ['No treatment'],
-            }),
-        ],
-        [
-            'outcome_of_treatment is submitted when treatment was unknown',
-            false,
-            loadObjects(treatment, {
-                outcome_of_treatment: 'treatment completed as prescribed',
-                treatment_type: ['Unknown'],
-            }),
-        ],
-    ],
-    response_to_treatment_criteria_method: [
-        [
-            'response_to_treatment_criteria_method is submitted when no treatment was given',
-            false,
-            loadObjects(treatment, {
-                response_to_treatment_criteria_method: 'RECIST',
-                treatment_type: ['no treatment'],
-            }),
-        ],
-        [
-            'response_to_treatment_criteria_method is missing when radiation treatment was given',
-            false,
-            loadObjects(treatment, {
-                treatment_type: ['radiation therapy'],
-            }),
-        ],
-    ],
+	treatment_start_interval: [
+		[
+			'treatment_start_interval is submitted when treatment was given',
+			true,
+			loadObjects(treatment, {
+				treatment_start_interval: 409,
+				treatment_type: ['Chemotherapy'],
+			}),
+		],
+		[
+			'treatment_start_interval is missing when treatment was given',
+			false,
+			loadObjects(treatment, {
+				treatment_type: ['Surgery'],
+			}),
+		],
+		[
+			'treatment_start_interval is submitted when no treatment was given',
+			false,
+			loadObjects(treatment, {
+				treatment_start_interval: 30,
+				treatment_type: ['No treatment'],
+			}),
+		],
+		[
+			'treatment_start_interval is submitted when treatment was unknown',
+			false,
+			loadObjects(treatment, {
+				treatment_start_interval: 0,
+				treatment_type: ['Unknown'],
+			}),
+		],
+		[
+			'treatment_start_interval is missing when no treatment was given',
+			true,
+			loadObjects(treatment, {
+				treatment_type: ['No treatment'],
+			}),
+		],
+		[
+			'treatment_start_interval is missing when treatment was unknown',
+			true,
+			loadObjects(treatment, {
+				treatment_type: ['Unknown'],
+			}),
+		],
+		[
+			'treatment_start_interval is submitted to be 0 when treatment was given',
+			true,
+			loadObjects(treatment, {
+				treatment_start_interval: 0,
+				treatment_type: ['Chemotherapy'],
+			}),
+		],
+		[
+			'treatment_start_interval is submitted to be a fraction when treatment was given',
+			true,
+			loadObjects(treatment, {
+				treatment_start_interval: 0.5,
+				treatment_type: ['Surgery'],
+			}),
+		],
+		[
+			'treatment_start_interval is submitted to be negative when treatment was given',
+			false,
+			loadObjects(treatment, {
+				treatment_start_interval: -0.2,
+				treatment_type: ['Ablation'],
+			}),
+		],
+	],
```

Comment:
a couple of additional tests to ensure even fractions pass, but not negative numbers.